Algorithm.CSharp/BasicTemplateAlgorithm.cs

```diff
@@ -55,7 +57,11 @@ public override void OnData(Slice data)
         {
             if (!Portfolio.Invested)
             {
-                SetHoldings(_spy, 1);
+                //SetHoldings(_spy, 1);
+                List<AlgoParams> AlgoParams = new List<AlgoParams>();
+                AlgoParams.Add(new AlgoParams("adaptivePriority", "Normal"));
+                SetHoldings(_spy, 1, false, "", OrderAlgorithm.AdaptiveAlgo, AlgoParams);
+                //MarketOrder(_spy, 100, true, "", OrderAlgorithm.AdaptiveAlgo, AlgoParams);
```

Comment:
Please revert changes to this file.

Reply:
Will do, this wasn't supposed to be committed.